### PR TITLE
update steps for api key, fix typo

### PR DESCRIPTION
**Pull request by ksh-b**
* added steps to make it easier to add api key/secret to script
* fixed some typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,17 +63,22 @@ logging.basicConfig(level=logging.DEBUG)
 
 ### Get api_secret
 api_secret is unique for each and every account. You need to enable api trading and get api_secret from alice blue.
-Please [contact](https://www.aliceblueonline.com/contact-us/) alice blue for getting api_secret.
+1. Please [contact](https://www.aliceblueonline.com/contact-us/) alice blue to get access to api.
+2. After you get a response from alice blue, login to [developer console](https://develop-api.aliceblueonline.com/dashboard)
+3. Click on 'Create App'
+4. Enter 'App Name' as you like. Enter 'Redirect URI' as 'http://develop-api.aliceblueonline.com/'
```
Comment (krishnavelu) [inline README.md]: Change this to `https://ant.aliceblueonline.com/plugin/callback`.
That is the default redirect_url of the library.

File: README.md [hunk 2]
```diff
@@ -63,17 +63,22 @@ logging.basicConfig(level=logging.DEBUG)
...
+6. Copy the 'App Id' and 'App Secret'. You will need these to generate access token.
 
 ### Get an access token
 1. Import alice_blue
 ```python
 from alice_blue import *
 ```
 
-2. Create access_token using login_and_get_access_token() function  with your `username`, `password`, `2FA` and `api_secret`
+2. Create access_token using login_and_get_access_token() function  with your `username`, `password`, `2FA`, `app_id` and `api_secret`. Both 2FA should be same.
```
Comment (krishnavelu) [inline README.md:79]: Add `If possible set both 2FA to 'a' as in example`

**Review by krishnavelu — CHANGES_REQUESTED** [review]
Didn't know I have these many typos. Thanks for correcting it.
Please create a pull request (after fixing my review comments) to develop branch, not directly to master branch.
I'll accept.

**Comment by ksh-b** [comment]
updated

**Comment by krishnavelu** [comment]
image

I don't want to see these commits in this PR.

File: README.md (hunk 2 above)
Comment (krishnavelu) [reply README.md:79]: I don't think this is resolved.

**Review by krishnavelu — CHANGES_REQUESTED** [review]

**Comment by krishnavelu** [comment]
@ksyko No issues. I've added all your changes along with review comments in this issue #285.
Closing this PR.

**Pull request closed without merging**